src/main/java/io/strimzi/kafka/bridge/http/HttpBridge.java

```diff
@@ -412,6 +412,12 @@ private void seekToEnd(RoutingContext routingContext) {
      * @param routingContext RoutingContext instance
      */
     private void processConsumer(RoutingContext routingContext) {
+
+        if (!bridgeConfig.getHttpConfig().isConsumerEnabled()) {
+            JsonObject message = new JsonObject();
+            message.put("message", "Consumer is disabled in config. To enable consumer update http.consumer.enabled to true");
+            HttpUtils.sendResponse(routingContext, HttpResponseStatus.OK.code(), BridgeContentType.JSON, message.toBuffer());
```

Comment:
I guess the method should return otherwise the code flow continues and, even if you already sent the HTTP response, the consumer will be created and then another it will try to send another HTTP response. Did that work for you? 
I was also wondering if a `503 Service Unavailable` is much better than a `200 OK`. @scholzj wdyt?